src/client/objectCard/ObjectCardView.js

```diff
@@ -61,7 +63,7 @@ const ObjectCardView = ({
     );
   };
   const objName = wObject.name || wObject.default_name;
-  const parentName = wObject.parent ? getFieldWithMaxWeight(wObject.parent, objectTypes.name) : '';
+  const parentName = parentObject.name || parentObject.default_name;
```

Comment:
How does card look in other components? Does it display parent from fields?

Reply:
++